### DISCUSSION TITLE
B-Hyve 4-Port XD (x6)

### DISCUSSION THREAD

**Discussion by drharris9121** [sebr/bhyve-home-assistant#119] [Ideas]
Thanks to @sebr for this amazing work.  I can confirm multiple 4-zone XDs work really well and the integration really solved a major headache.  I can now see all the zones on one screen, set them to rain delay as a group, manually water any zone, each with one click.  My irrigation dashboard is pretty rudimentary, so the next step is to expand the monitoring and functionality.  

![image](https://user-images.githubusercontent.com/96085772/166156119-e949df23-35f4-45a7-93a0-7e7e0b5e2edd.jpeg)

Instead of the existing manual on/off switches, manual timers for each.  A display of the entire system operating in real time (the indicators do light up yellow now when any zone is active).  Anyway, lots of ideas and I'm just starting to explore the hardware-agnostic irrigation control projects out there.

Before getting too far, I wanted to see if anyone here had ideas or would be willing to share.  So, what's on your irrigation dashboard?

**Comment by sebr** [comment]
@drharris9121 very cool, looking forward to hearing how you go with your projects!

**Reply by krusej23** [reply]
@drharris9121 What entity are you pulling for that history to show that way? All I am able to get is this type of look. Maybe I'm using the wrong card or something. I am using the logbook card.
<img width="500" alt="Screenshot 2023-10-10 at 8 58 39 PM" src="https://github.com/sebr/bhyve-home-assistant/assets/10549869/4cc79ac7-279d-4fbb-b1de-b21ac059b96e">
